test/k8s/datapath_configuration.go

```diff
@@ -361,6 +361,18 @@ var _ = Describe("K8sDatapathConfig", func() {
 			testIPMasqAgent()
 		})
 
+		It("DirectRouting, IPv4 only", func() {
```

Comment:
JFYI, we want to move this to CLI connectivity test in v1.15.

Reply:
OK thanks, this shouldn't be an issue I think? Or do you have an alternative to suggest?

Reply:
For this PR it's not an issue. I'm expecting the move to happen early in the v1.15 dev cycle.